theano/tensor/nnet/conv3d2d.py

```diff
@@ -338,7 +341,9 @@ def local_to_gpu(node):
                 new_inp = list(op_node.inputs)
                 for idx in to_gpu:
                     new_inp[idx] = cuda.gpu_from_host(new_inp[idx])
-                return [op()(*new_inp)]
+                new_node = op()(*new_inp)
+                copy_stack_trace(node.outputs[0], new_node)
```

Comment:
The stacktrace here should come from `host_input` (`node.inputs[0]`), which is the stack trace from when the actual `op` was instantiated, not when the `gpu_from_host` was.